lib/tessel/wifi.js

```diff
@@ -12,9 +16,7 @@ Tessel.prototype.findAvailableNetworks = function() {
 
         var resultsJSON = '';
 
-        remoteProcess.stderr.on('data', function(d) {
-          logs.err(d.toString());
-        });
+        remoteProcess.stderr.on('data', logErr);
```

Comment:
What we should do here (and below in `getWiFiInfo` is concatenate the stderr data as it comes in. Once the process is closed, we should check if the stderr data has any length, and if it does, reject the function with that string. See an example [here](https://github.com/tessel/t2-cli/blob/47219b6d28a6db9871ef12ff65203055a152edd4/lib/tessel/update.js#L24-L39). There are a couple of other points where we *don't* do this and we should start being more disciplined about it.